integration_tests/__tests__/modulenamemapper-test.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+'use strict';
+
+const runJest = require('../runJest');
+
+test('moduleNameMapper wrong configuration', () => {
+  const result = runJest('moduleNameMapper-wrong-config');
+  // Should fail because that wrong configuration will throw
+  expect(result.status).toBe(1);
```

Comment:
Can you also catch a snapshot if this error? It's great that we know it fails, but even better if we see what's thrown